homework.md

```diff
@@ -110,4 +113,4 @@ To record what changes you made to the code
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+The Git Cheatsheet! Stackoverflow etc.
```

Comment:
i think  Git Documentation website as well.

Reply:
Yep, I will add that one too. Thanks, alinosratipour